frontend/src/Stats.js

```diff
@@ -73,18 +76,11 @@ export const Stats = () => (
     errorPolicy="all"
   >
     {({ loading, error, data }) => {
-      if (loading) return ''
-      if (error)
-        return (
-          <P color="crimson" fontWeight="bolder">
-            Error connecting to database:
-            <br />
-            {error.message}
-          </P>
-        )
-
-      let { stats } = data
-      return <Chart data={stats.identifierFlaggingsWithin.summary} />
+      let summary = undefined
+      if (error) summary = demoData
```

Comment:
Nitpicky - should we specify the error being the error we get if not connected to the internet? Or do we want just any error leading to demo data right now?

Reply:
Huh, yeah, I like this idea 💯